preload/link-header-preload.html

```diff
@@ -6,10 +6,13 @@
     var t = async_test('Makes sure that Link headers preload resources');
 </script>
 <body>
-<script src="resources/dummy.js?pipe=trickle(d5)&link-header-preload"></script>
+<!--<script src="resources/dummy.js?pipe=trickle(d5)&link-header-preload"></script> -->
```

Comment:
Do we need to keep this comment in @yoavweiss?

Reply:
We really don't...